cocos/tween/actions/action-interval.ts

```diff
@@ -465,12 +467,12 @@ export function sequence (/* Multiple Arguments */tempArray: any): ActionInterva
         result = paramArray[0];
         for (let i = 1; i <= last; i++) {
             if (paramArray[i]) {
-                result = Sequence._actionOneTwo(result, paramArray[i]);
+                result = Sequence._actionOneTwo(result as ActionInterval, paramArray[i] as ActionInterval);
             }
         }
     }
 
-    return result;
+    return result as ActionInterval;
```

Comment:
Why not just  change the type of  `result` to `ActionInterval `?

Reply:
Because the input type is any, we need to assign the value.

Reply:
Can define it as ActionInterval instead of any.

Reply:
The input parameter is of type array, and the elements in the array are any.
This has to do with the type of the input array, and this needs more modification.

Reply:
What I mean is change 
```typescript
let result: any = null;
```
to 
```
let result: ActionInterval = null;
```

`result` is defined in this function, I think we can change it.

Reply:
ok